Tools/GNUMake/sites/Make.nersc

```diff
@@ -44,7 +44,20 @@ ifeq ($(which_computer),$(filter $(which_computer),perlmutter))
     endif
   endif
 
-  CUDA_ARCH = 80
+  ifeq ($(USE_CUDA),TRUE)
+    CUDA_ARCH = 80
+
+    ifneq ($(CUDA_ROOT),)
+        SYSTEM_CUDA_PATH := $(CUDA_ROOT)
+        COMPILE_CUDA_PATH := $(CUDA_ROOT)
+    else ifneq ($(CUDA_HOME),)
+        SYSTEM_CUDA_PATH := $(CUDA_HOME)
+        COMPILE_CUDA_PATH := $(CUDA_HOME)
+    else
+        $(error No CUDA_ROOT or CUDA_PATH found. Please load a cuda module.)
```

Comment:
`CUDA_PATH` or `CUDA_HOME`?

Reply:
All three are available in the module as identical paths, so I just added the third as well.

We can tweak later if they change or start pointing to different things.